test/string.js

```diff
@@ -176,6 +176,47 @@ describe('String types', () => {
       .equal(true);
   });
 
+  it('EMAIL is valid for the valid wikipedia examples', () => {
+    let v = string().email();
+
+    // https://en.wikipedia.org/wiki/Email_address#Examples
+    [
+      'simple@example.com',
+      'very.common@example.com',
+      'disposable.style.email.with+symbol@example.com',
+      'other.email-with-hyphen@example.com',
+      'fully-qualified-domain@example.com',
+      'user.name+tag+sorting@example.com',
+      'x@example.com',
+      'example-indeed@strange-example.com',
+      'admin@mailserver1',
+      'example@s.example',
+    ].map(email =>
+      v
+        .isValid(email)
+        .should.eventually()
+        .equal(true),
```

Comment:
@jquense seems like the pre-commit hook added an extra comma there, and I'm not sure if there are problems with that

my original code looked like this

```
    [
      'simple@example.com',
      'very.common@example.com',
      'disposable.style.email.with+symbol@example.com',
      'other.email-with-hyphen@example.com',
      'fully-qualified-domain@example.com',
      'user.name+tag+sorting@example.com',
      'x@example.com',
      'example-indeed@strange-example.com',
      'admin@mailserver1',
      'example@s.example'
    ].map(email => v.isValid(email).should.eventually().equal(true));
```

feel free to push adjustments anywhere if needed